components-core/src/main/java/com/adyen/checkout/components/model/payments/request/ConvenienceStoresJPPaymentMethod.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ozgur on 20/1/2023.
+ */
+
+package com.adyen.checkout.components.model.payments.request
+
+import com.adyen.checkout.core.exception.ModelSerializationException
+import com.adyen.checkout.core.model.getStringOrNull
+import kotlinx.parcelize.Parcelize
+import org.json.JSONException
+import org.json.JSONObject
+
+@Parcelize
+class ConvenienceStoresJPPaymentMethod(
+    override var type: String? = null,
+    override var firstName: String? = null,
+    override var lastName: String? = null,
+    override var telephoneNumber: String? = null,
+    override var shopperEmail: String? = null,
+) : EContextPaymentMethod() {
+
+    companion object {
```

Comment:
Can we add a `const val PAYMENT_METHOD_TYPE` to these classes? We also need to use them in `PaymentMethodDetails.getChildSerializer`